bin/dfx-software-sns-aggregator-version

```diff
@@ -20,17 +20,27 @@ source "$SOURCE_DIR/clap.bash"
 # Define options
 clap.define short=l long=pinned desc="Get the pinned version from versions.bash (default)" variable=GET_PINNED nargs=0
 clap.define short=l long=latest desc="Get the latest version" variable=GET_LATEST nargs=0
+clap.define short=m long=mainnet desc="Get the version on mainnet" variable=GET_MAINNET nargs=0
 # Source the output file ----------------------------------------------------------
 source "$(clap.build)"
 
 STRATEGY="${STRATEGY:-${GET_PINNED:+pinned}}"
 STRATEGY="${STRATEGY:-${GET_LATEST:+latest}}"
+STRATEGY="${STRATEGY:-${GET_MAINNET:+mainnet}}"
 STRATEGY="${STRATEGY:-pinned}" # Default - not guaranteed to be stable
 case "${STRATEGY}" in
 pinned)
   . "$SOURCE_DIR/versions.bash"
   echo "$SNS_AGGREGATOR_RELEASE"
   ;;
+mainnet)
+  prod_commit="$(git ls-remote git@github.com:dfinity/nns-dapp.git aggregator-prod | awk '{print $1}')"
```

Comment:
Note: Alternatively we could get the commit with:
```
 dfx canister metadata 3r4gx-wqaaa-aaaaq-aaaia-cai git_commit_id --network ic
```
This might be better but assumes that `dfx` is installed.  We usually have `dfx` installed so it's not much of a hurdle.